### ISSUE TITLE
Errors in logs

### ISSUE THREAD

**Issue by vavoluk** [akalongman/php-telegram-bot#30]
There are some errors in logs when I'm trying to use any command (excluding /echo):

```
[Fri Sep 04 15:29:57 2015] [error] [client 149.154.167.198] PHP Warning:  Missing argument 2 for Longman\\TelegramBot\\Entities\\Message::__construct(), called in /home/vavoluk/bot/vendor/longman/telegram-bot/src/Entities/Message.php on line 107 and defined in /home/vavoluk/bot/vendor/longman/telegram-bot/src/Entities/Message.php on line 64
[Fri Sep 04 15:29:57 2015] [error] [client 149.154.167.198] PHP Notice:  Undefined variable: bot_name in /home/vavoluk/bot/vendor/longman/telegram-bot/src/Entities/Message.php on line 66
```

How to fix this?

**Comment by akalongman** [comment]
Pass bot username in new Telegram

**Comment by vavoluk** [comment]
I already pass api key and botname:
```php
$API_KEY = 'MY_BOT_API_KEY';
$BOT_NAME = 'MY_BOT_NAME';
...
new Longman\TelegramBot\Telegram($API_KEY,$BOT_NAME)
...
```

**Comment by vavoluk** [comment]
Any idea how to fix this?

**Comment by MBoretto** [comment]
Thanks for reporting. Have you enable mysql or text logging? Could you
reproduce the error and report us logs and update?(correlated to each
other) I can't recognise the origin of the error with this information.

**Comment by vavoluk** [comment]
Yes, I have enabled both (mysql and text logging).
Have the same errors in error log:
```
[Sun Sep 06 16:14:30 2015] [error] [client 149.154.167.198] PHP Warning:  Missing argument 2 for Longman\\TelegramBot\\Entities\\Message::__construct(), called in /home/vavoluk/bot/vendor/longman/telegram-bot/src/Entities/Message.php on line 107 and defined in /home/vavoluk/bot/vendor/longman/telegram-bot/src/Entities/Message.php on line 64
[Sun Sep 06 16:14:30 2015] [error] [client 149.154.167.198] PHP Notice:  Undefined variable: bot_name in /home/vavoluk/bot/vendor/longman/telegram-bot/src/Entities/Message.php on line 66
```

**Comment by MBoretto** [comment]
It will be fixed in the next release

**Comment by MBoretto** [comment]
Fixed in 0.17
